packages/fmg-nitro-adjudicator/src/consensus-app/index.ts

```diff
@@ -11,94 +11,94 @@ import {
 import abi from 'web3-eth-abi';
 import { bigNumberify } from 'ethers/utils';
 
-export enum UpdateType {
-  Consensus,
-  Proposal,
-}
 export interface AppAttributes {
   furtherVotesRequired: Uint32;
   proposedAllocation: Uint256[];
   proposedDestination: Address[];
-  updateType: UpdateType;
-}
-export interface ProposalAppAttrs extends AppAttributes {
-  updateType: UpdateType.Proposal;
-}
-export interface ConsensusAppAttrs extends AppAttributes {
-  updateType: UpdateType.Consensus;
 }
-
 export interface ConsensusBaseCommitment extends BaseCommitment {
   appAttributes: AppAttributes;
 }
 
-export interface ProposalCommitment extends ConsensusBaseCommitment {
```

Comment:
Will this necessitate any changes in the wallet?

Reply:
For the wallet to use the newer version, yes!